tests/dynamic_data/publications/timeseries/timeseries_test.py

```diff
@@ -199,3 +200,18 @@ def test_timeseries_layer(layer: Publication, key, params, rest_method):
         asserts_publ.internal.correct_values_in_detail(layer.workspace, layer.type, layer.name,
                                                        **params.get('detail_values', {}),
                                                        )
+
+        for time in [
+            '2022-03-16T00:00:00Z',
+            '2022-03-19T00:00:00Z',
```

Comment:
Replace `:` in filenames e.g. with `-`, because `:` it's not allowed character in filename on Windows.